builtin/mainmenu/common.lua

```diff
@@ -14,31 +14,18 @@
 --You should have received a copy of the GNU Lesser General Public License along
 --with this program; if not, write to the Free Software Foundation, Inc.,
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
---------------------------------------------------------------------------------
 -- Global menu data
---------------------------------------------------------------------------------
 menudata = {}
 
---------------------------------------------------------------------------------
 -- Local cached values
---------------------------------------------------------------------------------
 local min_supp_proto, max_supp_proto
 
 function common_update_cached_supp_proto()
 	min_supp_proto = core.get_min_supp_proto()
 	max_supp_proto = core.get_max_supp_proto()
 end
 common_update_cached_supp_proto()
---------------------------------------------------------------------------------
 -- Menu helper functions
---------------------------------------------------------------------------------
-
---------------------------------------------------------------------------------
-local function render_client_count(n)
-	if     n > 99 then return '99+'
-	elseif n >= 0 then return tostring(n)
-	else return '?' end
-end
```

Comment:
But what if I specify the maximum client count number in a number with a hundred digits and break the table?

Reply:
Indeed, I should readd this.